readme.md

```diff
@@ -63,6 +63,7 @@ Document | Description
 [EC eHealth and COVID-19](https://ec.europa.eu/health/ehealth-digital-health-and-care/ehealth-and-covid-19_en) | Homepage of EU-DCC at the EC, here you'll find all of the officially published technical documentation as well as the various forms and procedures required for onboaring of non-EU countries (referred to as "Third Countries" in EC parlance)
 [Implementing decisions](https://ec.europa.eu/info/publications/commission-implementing-decisions-eu-equivalence-covid-19-certificates-issued-non-eu-countries_en) | Implementing decisions between the EU and Third Countries are published here.
 [Key documents](https://ec.europa.eu/info/publications/key-documents-related-digital-covid-19-certificate_en) | Key legal documents, ammendments etc
+[Revocation B2B](docs/revocation-b2b.pdf) | Documentation of the revocation B2B API from the gateway
```

Comment:
@ryanbnl Are we comfortable keeping copies of the `.pdf` files in several places? Should we link to the permanent place of the `.pdf` document? 

Do you suggest to move everything here?

Reply:
I'm going to move the technical documentation here. The versions on the EC website aren't always stored with version information and it keeps everything technical here.

Reply:
This implies we (=@ryanbnl) never got an answer to the question whether EC documents were archived somehow?

Reply:
We did, a negative.

Reply:
OK, so good to store it here as well.

Reply:
It's fine for me as well